modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,31 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by the name of the person who made the change, time/date and activity, and the saved state name depends on the activity you were doing at the time the save occurred.
```

Comment:
"The saved states are listed by the name of the person who made the change, time/date and activity, and the saved state name depends on the activity you were doing at the time the save occurred."
This is a run-on sentence.
Also, does the list really show names, or does it show user IDs?
Finally, translators have problems with items separated by forward slashes, such as "time/date" here. It's better to make them separate items in a list: time, date.

Reply:
Rewritten to address your comments.